packages/babel-plugin-object-enumerables/src/index.js

```diff
@@ -0,0 +1,19 @@
+const keysSource = "((o) => { var r = []; for (var k in o) { r.push(k) } return r; })";
```

Comment:
Yeah you could manually build the object with the babel-types methods like `t.identifier('a')` or use babel-template https://github.com/babel/babel/tree/master/packages/babel-template and then use `path.replaceWith`